client/src/nv_ingest_client/util/milvus.py

```diff
@@ -363,9 +363,9 @@ def _pull_text(element, enable_text: bool, enable_charts: bool, enable_tables: b
         pg_num = element["metadata"]["content_metadata"]["page_number"]
         doc_type = element["document_type"]
         if not verify_emb:
-            logger.error(f"failed to find embedding for entity: {source_name} page: {pg_num} type: {doc_type}")
+            logger.info(f"failed to find embedding for entity: {source_name} page: {pg_num} type: {doc_type}")
```

Comment:
Anything that get's logged to `info` would (should) have been logged by `error`. Is this trying to solve for missing error information?

Reply:
No this is so that we dont see the logs all the time. Currently in error they come up always. We got requests to make this less noisy.